constantine/src/main/java/org/hyperledger/besu/nativelib/constantine/LibConstantineVerkle.java

```diff
@@ -0,0 +1,113 @@
+package org.hyperledger.besu.nativelib.constantine;
+
+import com.sun.jna.Native;
+
+public class LibConstantineVerkle {
+    public static final boolean ENABLED;
+
+    static {
+        boolean enabled;
+        try {
+            Native.register(LibConstantineVerkle.class, "verkle");
+            enabled = true;
+        } catch (final Throwable t) {
+            t.printStackTrace();
+            enabled = false;
+        }
+        ENABLED = enabled;
+    }
+
+    // Declare native methods
+    public static native int ipa_commit(byte[] r, int r_len, byte[] inputs, int inputs_len);
```

Comment:
You might need to change the params of these functions according to current implementation. For example like this - https://github.com/hyperledger/besu-native/blob/d5504630332af07a22d519436dcfcbda2de3b134/ipa-multipoint/src/main/java/org/hyperledger/besu/nativelib/ipamultipoint/LibIpaMultipoint.java#L52

Reply:
yes for verkle we need all the methods listed in LibIpaMulitpoint 

you can ignore verifyPreStateRoot for the moment